sdk/webpubsub/Azure.Messaging.WebPubSub.Client/tests/Azure.Messaging.WebPubSub.Client.Tests.csproj

```diff
@@ -11,7 +11,6 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Azure.Messaging.WebPubSub" />
   </ItemGroup>
-
   <ItemGroup Condition="$(TargetFramework) != 'net461'">
```

Comment:
This condition needs to be removed.   Tests must run and pass on all of the frameworks supported by Azure SDK packages.

Reply:
Will be removed.